modules/nw-networkpolicy-allow-external-clients.adoc

```diff
@@ -0,0 +1,100 @@
+// Module included in the following assemblies:
+//
+// * networking/multiple_networks/configuring-multi-network-policy.adoc
+
+:_content-type: PROCEDURE
+[id="nw-networkpolicy-allow-external-clients_{context}"]
+= Creating a multi-network policy to allow traffic from external clients
+
+
+[NOTE]
+====
+If you log in with a user with the `cluster-admin` role, then you can create a network policy in any namespace in the cluster.
+====
+
+Follow this procedure to configure a policy that allows external service from the public Internet directly or through a Load Balancer to access the pod. Traffic is only allowed to a pod with the label `app=web`.
+
+.Prerequisites
+
+* Your cluster uses a cluster network provider that supports `NetworkPolicy` objects, such as
+ifndef::ovn[]
+the OpenShift SDN network provider with `mode: NetworkPolicy` set.
+endif::ovn[]
+ifdef::ovn[]
+the OVN-Kubernetes network provider or the OpenShift SDN network provider with `mode: NetworkPolicy` set.
+endif::ovn[]
+This mode is the default for OpenShift SDN.
+* You installed the OpenShift CLI (`oc`).
+* You are logged in to the cluster with a user with `{role}` privileges.
+* You are working in the namespace that the {name} policy applies to.
+
+.Procedure
+
+. Create the following YAML that defines a `deny-by-default` policy to deny ingress from all pods in all namespaces. Save the YAML in the `deny-by-default.yaml` file:
++
+This is a fundamental policy, blocking all cross-pod networking other than cross-pod traffic allowed by the configuration of other network policies.
++
+[source,yaml]
+----
+apiVersion: k8s.cni.cncf.io/v1beta1
+kind: MultiNetworkPolicy
+metadata:
+  name: deny-by-default
+  namespace: default <1>
+  annotations:
+    k8s.v1.cni.cncf.io/policy-for: <network_name> <2>
+spec:
+  podSelector: {} <3>
+  ingress: [] <4>
+----
+<1> `namespace: default` deploy this policy to the `default` namespace.
+<2> `<network_name>`:: Specifies the name of a network attachment definition.
+<3> `podSelector:` is empty, this means it will match all the pods. Therefore, the policy will be enforced to all pods in the default namespace.
+<4> There are no `ingress` rules specified. This causes incoming traffic to be dropped to the selected (=all) pods.
+
+. Apply the policy by entering the following command:
++
+[source,terminal]
+----
+$ oc apply -f deny-by-default.yaml
+----
++
+.Example output
+[source,terminal]
+----
+multinetworkpolicy.k8s.cni.cncf.io/deny-by-default created
+----
+
+. Create a policy that allows traffic from the public Internet directly or through a load balancer to access the pod.
++
+This policy ensures traffic from an external service is only allowed to a pod with the label `app=web`. Save the YAML in the `web-allow-external.yaml` file:
++
+[source,yaml]
+----
+apiVersion: k8s.cni.cncf.io/v1beta1
```

Comment:
This policy is not going to work as expected due to the bug:
[bug](https://issues.redhat.com/browse/OCPBUGS-974)
You have to add  the PolicyType field

Reply:
I was pulling my hair out, wondering why it did not work. Hmm might leave out - thinking